redwood-widget-compose/build.gradle

```diff
@@ -17,5 +17,11 @@ kotlin {
         api libs.jetbrains.compose.runtime
       }
     }
+    commonTest {
+      dependencies {
+        implementation libs.kotlinx.coroutines.core
```

Comment:
We shouldn't need this. Seems like Compose multiplatform isn't correctly exposing coroutines as an `api` dependency. I may look into it... or I may not. Wasn't worth a "TODO".